cmd/runtimetest/main.go

```diff
@@ -695,7 +697,12 @@ func main() {
 		cli.StringFlag{
 			Name:  "path",
 			Value: ".",
-			Usage: "path to the configuration",
+			Usage: "Path to the configuration",
+		},
+		cli.StringFlag{
+			Name:  "platform",
```

Comment:
@Mashimiao We can, but can we do that in a follow up PR, what we did in this PR looks consistent and we have the default value.

@liangchenye runc doesn't check platform at all, the spec author should be responsible to give runc the right config.